lib/git_reflow/git_server/git_hub/pull_request.rb

```diff
@@ -78,6 +78,65 @@ def approved?
           end
         end
 
+        def merge!(options = {})
+
+          # fallback to default merge process if user "forces" merge
+          if(options[:skip_lgtm])
+            super options
+          else
+            say "Merging pull request ##{self.number}: '#{self.title}', from '#{self.feature_branch_name}' into '#{self.base_branch_name}'"
+
+            options[:base] ||= 'master'
+            feature_branch_name = GitReflow.current_branch
+
+            unless options[:title] || options[:message]
+
+              # prompts user for commit_title and commit_message
+              pull_request_msg_file = "#{GitReflow.git_root_dir}/.git/SQUASH_MSG"
+              default_editor        = "#{ENV['EDITOR']}"
```

Comment:
This should move to a constant (now that we have a need to reference it in more than one place).  Put this in the main `lib/git_reflow.rb` file:

```ruby
DEFAULT_EDITOR = "#{ENV['EDITOR']}".freeze || "vi".freeze
```